src/transport.rs

```diff
@@ -216,19 +221,10 @@ implement_http_transport! {
         signal: Arc<Condvar>,
         shutdown_immediately: Arc<AtomicBool>,
         queue_size: Arc<Mutex<usize>>,
+        http_client: Arc<Client>,
```

Comment:
Just a random note… why `Arc`? `Client` is cheap to clone and shares the background thread, so there's no real reason to add another layer of indirection.

Reply:
Right, it isn't necessary with this trick, thanks.

```
sentry::ClientOptions {
        transport: Box::new(|options: &sentry::ClientOptions| -> Box<sentry::internals::Transport> {
            Box::new(ReqwestHttpTransport::new(options, Some(reqwest::Client::clone(&arc_client))))
        }),
        // ...
}
```